---
layout: archive
title: ""
permalink: /publications/
author_profile: true
---

Research Interests
======
I am broadly interested in developing Bayesian methods that are computationally efficient and have theoretical guarantees. My recent work is focused on developing flexible Bayesian methods and efficient computational algorithms for big data sets, tailored for both their complexity and size. Motivating examples include big data in genomics, geostatistics, and biomedical databases. Simultaneously optimizing for the size and complexity is a challenge with current Bayesian methods. I am developing novel and computationally tractable Bayesian methods using principles from machine learning and asymptotic Bayesian statistics. 

My current research work has three major themes: 
* developing sampling algorithms using divide-and-conquer techniques and asynchronous computations for scaling Bayesian inference;  
* developing parametric and nonparametric models for analyzing electronic health records data, where the covariates have non-linear dependence structures such as grammar and graph-based hierarchy; and 
* developing array-variate models for modeling multimodal data from local field potentials, brain imaging, and gene expression.

You can find my articles on [Google Scholar](https://scholar.google.com/citations?user=BR_89wIAAAAJ&hl=en). 

You can find the code for most of the manuscripts at [Github](https://github.com/blayes). 

Publications (Published or Under Review)
======

1. Winter, S., Campbell, T., Lin, L., Srivastava, S., Dunson, D.B. Machine Learning and the Future of Bayesian Computation. Under review at Statistical Science. [Arxiv](https://arxiv.org/abs/2304.11251).
3. Savitsky, T., Williams, M., Srivastava, S. Pseudo Bayesian estimation of one-way ANOVA model in complex surveys. Submitted to Survey Methodology. [Arxiv](https://arxiv.org/abs/2004.06191). 
2. Wang, C. and Srivastava, S. Asymptotic Normality of the Posterior Distributions in a Class of Hidden Markov Models. Under revision. [Arxiv](https://arxiv.org/abs/2105.14394).
1. Hing, B., Mitchell, S.B, Eberle, M., Filali, Y., Matkovich, M., Kasturirangan, M., Hultman, I., Wyche, W., Jimenez, A., Velamuri, R., Johnson, M., Srivastava, S., Hultman, R. Transcriptomic Evaluation of Stress Vulnerability Network using Single Cell RNA-Seq in mouse Prefrontal Cortex. Under review. [bioRxiv](https://www.biorxiv.org/content/10.1101/2023.05.14.540705v1.abstract).
1. Schaffler, M.D., Johnson, M., Hing, B., Kahler, P., Hultman, I., Srivastava, S., Arnold, J., Blendy, J.N., Hultman, R., Abdus-Saboor, I. A critical role for touch neurons in a skin-brain pathway for stress resilience. Under review. [Biorxiv](https://www.biorxiv.org/content/10.1101/2022.05.23.493062v1). 
1. Srivastava, S., Xu, Z., Li, Y., Street, N., Gilbertson-White, S. Gaussian process regression and classification using International Classification of Disease codes as covariates. Under revision.  [Arxiv](http://arxiv.org/abs/2108.01813) [Code](https://github.com/blayes/ehr-data-analysis). Accepted for publication in STAT.
1. Wang, C. and Srivastava, S. Divide-and-conquer Bayesian inference in hidden Markov models (2023). Electronic Journal of Statistics. [Arxiv](https://arxiv.org/abs/2105.14395). 
1. Zhou, J, Khare, K, and Srivastava, S. Asynchronous and Distributed Data Augmentation for Massive Data Settings (2022). Journal of Computational and Graphical Statistics. [Arxiv](https://arxiv.org/abs/2109.08969) [Code](https://github.com/blayes/ADDA). 
2. Guhaniyogi, R., Li, C., Savitsky, T., Srivastava, S. Distributed Bayesian varying coefficients modeling using a Gaussian process prior (2022). Journal of Machine Learning Research. [Arxiv](https://arxiv.org/abs/2006.00783). 
2. Shyamalkumar, N.D., Srivastava, S. An algorithm for Distributed Bayesian Inference (2022). STAT [Arxiv](https://arxiv.org/abs/1911.07947) [Code](https://github.com/blayes/location-scatter-wasp). 
3. Guhaniyogi, R., Li, C., Savitsky, T., Srivastava, S. A divide-and-conquer Bayesian approach to large-scale kriging. Major revision requested by Statistical Science. [Arxiv](https://arxiv.org/abs/1712.09767) [Code](https://github.com/blayes/LSWASP).
4. Xu, Y. and Srivastava, S. (2021). Distributed Bayesian Inference in Linear Mixed-Effects Models. Journal of Computational and Graphical Statistics. [Link](https://www.tandfonline.com/doi/abs/10.1080/10618600.2020.1869025). 
5. Yao, H., Srivastava, S., Swyers, N. C., Han, F., Doerge, R. W., Birchler, J. A. (2020). Inbreeding depression in genotypically matched diploid and tetraploid maize. Frontiers in Genetics. [Link](https://www.frontiersin.org/articles/10.3389/fgene.2020.564928/full).
6. Kandemirli, S. G., Priya, S., Chopra, S., Ward, C., Locke, T., Soni, N., Srivastava, S., Jones, K., Bathla, G. (2020). Presurgical detection of brain invasion status in meningiomas based on first-order histogram based texture analysis of contrast enhanced imaging. Clinical Neurology and Neurosurgery. [Link](https://europepmc.org/article/med/32932028).
7. Bathla, G., Derdeyn, C. P., Moritani, T., Freeman, C. W., Srivastava, S., Song, J., Soni, N. (2020). Retrospective, dual-center review of imaging findings in neurosarcoidosis at presentation: prevalence and imaging sub-types. Clinical Radiology. [Link](https://pubmed.ncbi.nlm.nih.gov/32703543/).
8. Srivastava, S., DePalma, G., Liu, C. (2019). Distributed Expectation-Maximization algorithm for massive data: The DEM algorithm. Journal of Computational and Graphical Statistics. [Arxiv](https://arxiv.org/abs/1806.07533) [Code](https://github.com/blayes/DEM).
9. Srivastava, S., Li, C., Dunson, D. B. (2018). Scalable Bayes via barycenter in Wasserstein space. Journal of Machine Learning Research. [Link](https://jmlr.org/papers/v19/17-084.html) [Code](https://github.com/blayes/WASP).
10. Savitsky, T. D. and Srivastava, S. (2018). Scalable Bayes under informative sampling. Scandinavian Journal of Statistics. [Arxiv](https://arxiv.org/abs/1606.07488).
11. Minsker, S., Srivastava, S., Lin, L., Dunson, D. B. (2017). Robust and scalable Bayes via a median of subset posterior measures. Journal of Machine Learning Research. [Link](https://jmlr.org/papers/v18/16-655.html) [Code](https://github.com/blayes/Mposterior).
12. Schaich Borg, J., Srivastava, S., Lin, L., Dunson, D. B., Dziraza, K., de Lecea, L. (2017). Anterior cingulate and insula sub-networks encode intersubjective avoidance in rats. Brain and Behavior. [Link](https://onlinelibrary.wiley.com/doi/full/10.1002/brb3.710).
13. Srivastava, S., Engelhardt, B. E., Dunson, D. B. (2017). Expandable factor analysis. Biometrika. [Arxiv](https://arxiv.org/abs/1407.1158) [Code](https://github.com/blayes/xfa).
14. Li, C., Srivastava, S., Dunson, D. B. (2017). PIE: simple, scalable and accurate posterior interval estimation. Biometrika. [Arxiv](https://arxiv.org/abs/1605.04029) [Code](https://github.com/blayes/PIE).
15. Srivastava, S., Cevher, V., Tran-Dinh, Q., Dunson, D. B. (2015). WASP: Scalable Bayes via barycenter of subset posteriors. 18th International Conference on Artificial Intelligence and Statistics (AISTATS), San Diego, USA. 
16. Minsker, S., Srivastava, S., Lin, L., Dunson, D. B. (2014). Robust and scalable Bayes via the median posterior. 31st International Conference on Machine Learning (ICML), Beijing, China.
12.	Johnson, A. J., Shukle, R. H., Chen, M.-S., Srivastava, S., O. (2015). Differential expression of candidate salivary effector proteins in field collections of Hessian fly, Mayetiola destructor. [Link](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4406158/).
13.	Nagarajan, S., Srivastava, S., Sherman, L. A. (2014). Essential role of the plasmid hik31 operon in regulating central metabolism in the dark in Synechocystis sp. PCC 6803. Molecular microbiology. [Link](https://pubmed.ncbi.nlm.nih.gov/24237382/).
14.	Srivastava, S., Wang, W., Manyam, G., Ordonez, C., Baladandayuthapani, V. (2013). Integrating multi-platform genomic data using hierarchical Bayesian relevance kernel machines. EURASIP Journal on Bioinformatics and Systems Biology. [Link](https://pubmed.ncbi.nlm.nih.gov/23809014/).
15.	Auer, P. L., Srivastava, S., Doerge, R. W. (2012). Differential expression--the next generation and beyond. Briefings in Functional Genomics. [Link](https://academic.oup.com/bfg/article/11/1/57/191815).

Funding
------

Most of the research work is supported in part by grants from the Office of Naval Research and National Science Foundation. 


